juntagrico/templates/size_change.html

```diff
@@ -5,34 +5,74 @@
 {% block page_title %}
     {% vocabulary "subscription" as v_subscription %}
     <h3>
-        {% blocktrans %}{{ v_subscription }}-Grösse ändern{% endblocktrans %}
+        {% blocktrans %}{{ v_subscription }}-Bestandteile ändern{% endblocktrans %}
     </h3>
 {% endblock %}
 {% block content %}
     {% config "enable_shares" as c_enable_shares %}
     {% vocabulary "subscription" as v_subscription %}
     {% vocabulary "subscription_pl" as v_subscription_pl %}
     {% vocabulary "share_pl" as v_share_pl %}
-    {% if saved %}
-        <div class="alert alert-success">
-            {% blocktrans %}Die {{ v_subscription }}-Grösse wurde erfolgreich geändert.{% endblocktrans %}
-            <br/>
-            <a href="{% url 'sub-detail' %}" class="alert-link">
-                {% blocktrans %}Zurück zur {{ v_subscription }}-Übersicht{% endblocktrans %}
-            </a>
+
+
+    <div class="row mb-3">
+        <b>{% trans "Bestandteile verwalten" %}</b>
+    </div>
+    <div class="row">
+            <table id="filter-table" class="table" style="display: table;">
+                <thead>
+                    <tr>
+                        <th>
+                            {% trans "Grösse" %}
+                        </th>
+                        <th>
+                            {% trans "Status" %}
+                        </th>
+                        <th>
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for part in subscription.parts.all %}
+                        <tr>
+                            <td>
+                                {{ part.type.size.name }}
+                            </td>
+                            <td>
+                                {{ part.state_text }}
+                            </td>
+                            <td>
+                                {% if not part.canceled and part.deactivation_date is None and part.can_cancel %}
+                                    <a href="{% url 'part-cancel' part.id subscription.id %}"class="btn btn-success">
```

Comment:
At least a confirmation dialog should be added to prevent accidental cancelation.